atomate/vasp/firetasks/glue_tasks.py

```diff
@@ -202,6 +203,52 @@ def run_task(self, fw_spec):
         return FWAction(stored_data=stored_data)
 
 
+@explicit_serialize
+class GetInterpolatedPOSCAR(FiretaskBase):
+    """
+    Grabs CONTCARS from two previous calculations to create interpolated
+    structure.
+
+    Required params:
+        start (str): name of fw for start of interpolation.
+        end (str): name of fw for end of interpolation.
+        this_image (int): which interpolation this is.
+        nimages (int) : number of interpolations.
+        autosort_tol (float): a distance tolerance in angstrom in which
+          to automatically sort end_structure to match to the closest
+          points in this particular structure.
+    """
+    required_params = ["start","end","this_image","nimages","autosort_tol"]
+
+    def run_task(self, fw_spec):
+
+        # make folder for poscar interpolation start and end structure files.
+        interpolate_folder = '/interpolate'
+        if not os.path.exists(os.getcwd()+interpolate_folder):
```

Comment:
use os.path.join(), not string concatenation. You also don't need slashes in that case, e.g. don't need "/interpolate" just "interpolate"

Reply:
(this comment is throughout the code)

Reply:
Done.